array/DNAm/preprocessing/CETYGOdeconvolution.r

```diff
@@ -0,0 +1,195 @@
+##---------------------------------------------------------------------#
+##
+## Title: Run CETYGO cell type deconvolution
+##
+## Purpose of script: Use CETYGO brain ref panal to run CETYGO on the samples
+##
+## Author: Emma Walker
+##
+## Date Created: Feb 2024
+##
+##---------------------------------------------------------------------#
+
+
+#----------------------------------------------------------------------#
+# NOTES
+#----------------------------------------------------------------------#
+# project folder is provided on command line
+# More info on the CETYGO package can be found here: 
+#   https://github.com/ejh243/CETYGO/wiki/Deconvolution-of-brain-cell-types
+
+
+#----------------------------------------------------------------------#
+# DEFINE PARAMETERS
+#----------------------------------------------------------------------#
+
+args<-commandArgs(trailingOnly = TRUE)
+dataDir <- args[1]
+
+gdsFile <-paste0(dataDir, "/2_gds/raw.gds")
+configFile <- paste0(dataDir, "/config.r")
+
+gdsObj<-ifelse(file.exists(gdsFile), TRUE, ifelse(file.exists(msetFile), FALSE, NA))
+
+source(configFile)
+
+
+#----------------------------------------------------------------------#
+# LOAD PACKAGES
+#----------------------------------------------------------------------#
+
+library(CETYGO)
```

Comment:
Do we need to add this library to the installLibraries.r script?

Reply:
done